backend/api/Areas/Notes/Controllers/NoteController.cs

```diff
@@ -59,6 +61,41 @@ public IActionResult AddNote(NoteType type, [FromBody] EntityNoteModel noteModel
             var createdNote = _noteService.Add(type, noteModel);
             return new JsonResult(createdNote);
         }
+
+        /// <summary>
+        /// Get the notes for the specified type.
+        /// </summary>
+        /// <param name="type">Used to identify note type.</param>
+        /// <returns></returns>
+        [HttpGet("{type}")]
+        [Produces("application/json")]
+        [HasPermission(Permissions.NoteView)]
+        [ProducesResponseType(typeof(IEnumerable<NoteModel>), 200)]
+        [SwaggerOperation(Tags = new[] { "note" })]
+        public IActionResult GetNotes(NoteType type)
+        {
+            var notes = _noteService.GetNotes(type);
+            var mappedNotes = _mapper.Map<List<NoteModel>>(notes);
+            return new JsonResult(mappedNotes);
+        }
```

Comment:
@Maheshbabu28 this would return ALL notes of type "activity", not just the ones for a given ActivityID
I think the notes feature requires a type and the parentId to retrieve the associated notes? Can you confirm that getting all notes from the database is what is desired here?

Reply:
This is what I could find on the story acceptance criteria:
* Users should be able to see list of notes **added to the entity** in a tabular format

So we would need to pass the entityId to this endpoint to filter the notes